Tasks/JavaToolInstallerV0/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -53,5 +53,15 @@
   "loc.messages.CorrelationIdForARM": "Correlation ID from ARM api call response : %s",
   "loc.messages.JavaNotPreinstalled": "Java %s is not preinstalled on this agent",
   "loc.messages.UsePreinstalledJava": "Use preinstalled JDK from %s",
-  "loc.messages.WrongArchiveStructure": "JDK file is not valid. Verify if JDK file contains only one root folder with 'bin' inside."
+  "loc.messages.WrongArchiveStructure": "JDK file is not valid. Verify if JDK file contains only one root folder with 'bin' inside.",
+  "loc.messages.NewJDKIsNotInstalled": "New JDK is not installed.",
```

Comment:
Is this used anywhere? I'm not seeing it but could just be missing it

Reply:
Thanks! Fixed it